NBISweden/urdr
# Dev/login design

## Related issue(s) and PR(s)
This PR closes #134 .



## Type of change

 
## List of changes made

- the login page got a new design
- the information to use Redmine credentials was added together with a link to Redmine
- a config file for ste CSS linter "stylelint" was added to allow legacy color notations. This was needed to make the box shadow work without the linter complaining.
- the `basic-button` class was moved in the stylesheet so that other, more specific button classes that are defined later will override the styles from `basic-button` if two classes are used simultaneously.

## Screenshot of the fix


## Testing

- open urdr without being logged in and see the new styling
- try logging in with wrong credentials to see that the login error still works and looks nice.
- log in and make sure nothing has changed on the `/report` page. Pay special attention to the buttons.

## Further comments


## Definition of Done checklist
